### PR TITLE
Cringe 101 workshop

### PR DESCRIPTION
**Pull request by MaxWofford**
***DO NOT MERGE***

This is the warped hellspawn of html and markdown that should never have been born unto this world. DO NOT put it on master until it has been fixed.

### CODE REVIEW

**Comment by JevinSidhu** [comment]
@MaxWofford Thanks for helping out with this :) It turned out much better than I thought it'd look. :100:

**Comment by MaxWofford** [comment]
@JevinSidhu Could I get links to the google docs in this workshop? Dropbox doesn't show any information about the drive links.

**Comment by JevinSidhu** [comment]
@MaxWofford 

Link: https://drive.google.com/folderview?id=0BxDM7p7t9f-GczV3N0pZTHJmUTQ&usp=sharing

**Comment by jonleung** [comment]
@MaxWofford were you working on this further?

**Comment by jonleung** [comment]
Just wondering because the tests are currently failed for this.

**Comment by MaxWofford** [comment]
That should fix all the markcop tests. @hackclub/hackclub Anyone want to review this and merge it?

**Comment by paked** [comment]
I've quickly gone through and found formatting errors.

Have not reviewed the content so much.

**Comment by paked** [comment]
I'll do this in a few hours, leaving unassigned so that someone else can claim if they want to :)

File: workshops/cringe-101/README.md [hunk 1]
```diff
@@ -0,0 +1,633 @@
...
+  - img is the tag name → This is self-closing, meaning there is no a closing
+    tag like </img>
+  - src is an attribute that specifies the URL of the image
+    - Think of it like a setting
+  - Add the URL (redirected to image) between the quotes
+
+http://imgur.com/ is a website where you can upload and then get a link to the
+image
+
+- HTML is used for content
```
Comment (paked) [inline workshops/cringe-101/README.md]: is this correct?

**Comment by MaxWofford** [comment]
Blocking on https://github.com/hackclub/hackclub/pull/540

**Comment by paked** [comment]
Taking this off of @MaxWofford's plate so we can get it merged in tonight!

**Comment by paked** [comment]
Closing in favour of #577

**Pull request closed without merging**